backend/src/main/java/com/summerHack/diningTogether/controller/FoodController.java

```diff
@@ -0,0 +1,54 @@
+package com.summerHack.diningTogether.controller;
+
+import com.summerHack.diningTogether.model.Food;
+import com.summerHack.diningTogether.model.FoodBrief;
+import com.summerHack.diningTogether.service.FoodService;
+import lombok.AllArgsConstructor;
+import org.springframework.beans.factory.annotation.Autowired;
+
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+
+@AllArgsConstructor
+@RestController
+@RequestMapping("/food")
+public class FoodController {
+    @Autowired
+    private FoodService foodService;
+
+    @GetMapping("/all")
```

Comment:
To get food by category, you can simply use url `GET /api/v1/foods?category=...`. If user does not include an parameter in the url, all food list is returned.

BTW, since we only have limited time, you don't have to implement pagination this time.